libs/lib-client/diagram/src/lib/diagram/diagram-row.component.ts

```diff
@@ -26,6 +26,7 @@ import { DragTileService, TilesGroupedByZone } from '../drag-tiles';
 })
 export class DiagramRowComponent implements OnChanges {
   @Input() row: Tile[];
+  @Input() rowParents: string[];
```

Comment:
We can reduce the dependency of adding a new input to the diagram-row component from it's parent. We can maintain the lookup for each row or a parent tile in the drag-tile service rather than depend on diagram component provide this detail.

Reply:
okay, removed property binding